src/main/java/de/tum/in/www1/artemis/web/rest/plagiarism/PlagiarismResource.java

```diff
@@ -184,4 +183,19 @@ public ResponseEntity<Void> deletePlagiarismComparisons(@PathVariable("exerciseI
         }
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * Counts the number of plagiarism results for the given exercise
+     *
+     * @param exerciseId the id of the exercise
+     * @return the number of plagiarism results
+     */
+    @GetMapping("exercises/{exerciseId}/plagiarism-results-count")
+    @EnforceAtLeastInstructor
+    public long getNumberOfPlagiarismResultsForExercise(@PathVariable("exerciseId") long exerciseId) {
+        var exercise = exerciseRepository.findByIdElseThrow(exerciseId);
+        authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.INSTRUCTOR, exercise, null);
+        // every result can lead to two plagiarism cases, that's why we multiply by two
```

Comment:
Couldn't it also be three? What happens if three people plagarize each other?

Reply:
Three different cases, see my screenshot.
![image](https://github.com/ls1intum/Artemis/assets/84102468/41381abd-eac4-43ca-8130-9679b1f11ed7)

Reply:
Yes, but in the overview it will show 4:

![screenshot-2023-08-08_001761](https://github.com/ls1intum/Artemis/assets/9006596/24f28e1b-0049-4d0d-b950-85f46db95248)

This is wrong I think: I only have 3 submissions.

Reply:
Also, the overview says "1 suspicious sessions", but I logged in with three users (using the same browser window). Isn't that three sessions?

Reply:
regarding the second point, I think the wording should be improved but the counting is correct because we count suspicious cases and you have one case to which 3 sessions belong.

Reply:
the first aspect should be fixed now.